README.rst

```diff
@@ -134,7 +134,11 @@ From the ``ckanext-showcase`` directory::
     paster showcase migrate -c {path to production.ini}
 
 Note that each Related Item must have a unique title before migration can
-proceed.
+proceed. If you prefer resolving duplicates as showcases, you can use the force
+option to migrate them anyways. Duplicate Relations will be created as
+'duplicate_' + original_related_title + '_' + hash
+
+    paster showcase migrate -c {path to production.ini} force
```

Comment:
I'd prefer `force` to be a command line option, with a more descriptive name, such as `--allow-duplicates`. ckanext-harvester has an [example](https://github.com/ckan/ckanext-harvest/blob/master/ckanext/harvest/commands/harvester.py#L129) of how to create cli options.